.github/workflows/continuous-integration.yml

```diff
@@ -72,8 +72,10 @@ jobs:
           extensions: "${{ matrix.php-extensions }}"
           ini-values: "zend.assertions=1, max_execution_time=30"
 
-      - name: "Install symfony/flex"
-        run: "composer global require --no-progress --no-scripts --no-plugins symfony/flex"
+#      - name: "Install symfony/flex"
+#        run: |
+#          "composer global config --no-plugins allow-plugins.symfony/flex true"
+#          "composer global require --no-progress --no-scripts --no-interaction --no-plugins symfony/flex"
```

Comment:
@ostrolucky can we remove this CI step or is necessary? It is causing some problems ([error](https://github.com/snc/SncRedisBundle/runs/7279992495?check_suite_focus=true#step:11:23))

Reply:
It's necessary to make SYMFONY_REQUIRE env var working. We need to find out why this stopped working

Reply:
I've noticed that you have just fixed on branch master the composer error. 
FYI if you don't want to modify `composer.json` it works also with these [changes](https://github.com/snc/SncRedisBundle/pull/669/commits/214f7990f92c2f6d5cdf4b13eaad1213cc3f1cf9)

Reply:
I think it's better to modify composer.json because that's where we already allowed another plugin too. Can you rebase your branch on master?

Reply:
Done :)